### ISSUE TITLE
Unable to carry out the steps under "Re-train CoLAKE" section of README.md

### ISSUE THREAD

**Issue by calebjacksonhoward** [txsun1997/CoLAKE#13]
Specifically, the step:
python preprocess/gen_data.py 4

Fails with:
Exception has occurred: KeyError
'ents'
  File "/home/caleb/Work/CareActr/projects/phase1/repo/CoLAKE/preprocess/gen_data.py", line 62, in load_data
    h, t = ins['ents'][0][0], ins['ents'][1][0]
  File "/home/caleb/Work/CareActr/projects/phase1/repo/CoLAKE/preprocess/gen_data.py", line 95, in <module>
    d_ent, head_cluster = load_data()

**Comment by txsun1997** [comment]
Sorry for the late response. It seems that the data '../ernie_data/fewrel/test.json' from https://github.com/thunlp/ERNIE has changed. Fortunately, as commented in the code, this part is to remove FewRel test set from our training data for a fair comparison. So if you do not aim to publish a paper, you can just discard this part of codes: line #54 - line #61.

**Comment by calebjacksonhoward** [comment]
Thank you!
